Yoast/YoastSEO.js
# Add internal linking for Russian

## Summary

This PR can be summarized in the following changelog entry: Adds internal linking for Russian

## Relevant technical choices:



## Test instructions

This PR can be tested by following these steps:

- Use the browserified relevant words example.
- Set language to ru_RU.
- Take a Russian-language text (e.g., from wiki).
- Make sure words from the list of function words do not appear in the table.

Fixes #1455
